vllm/model_executor/models/deepseek_mtp.py

```diff
@@ -158,14 +158,13 @@ def forward(
         self,
         input_ids: torch.Tensor,
         positions: torch.Tensor,
-        previous_hidden_states: torch.Tensor,
+        hidden_states: torch.Tensor,
```

Comment:
NOTE: This fixes a critical bug breaking MTP support, since the arguments are now passed as `kwargs` by `eagle.py` and therefore must be called `hidden_states`.